vms/platformvm/block/builder/helpers_test.go

```diff
@@ -428,3 +429,17 @@ func shutdownEnvironment(env *environment) error {
 		env.baseDB.Close(),
 	)
 }
+
+func getValidTx(txBuilder txbuilder.Builder, t *testing.T) *txs.Tx {
```

Comment:
This is used in `builder_test.go` and was originally in `network_test.go`. Added here as we moved `network_test`.go to its own pkg